lighthouse-core/computed/page-dependency-graph.js

```diff
@@ -144,16 +144,19 @@ class PageDependencyGraph {
    */
   static linkNetworkNodes(rootNode, networkNodeOutput) {
     networkNodeOutput.nodes.forEach(node => {
+      const directInitiatorRequest = node.record.initiatorRequest || rootNode.record;
+      const directInitiatorNode =
+        networkNodeOutput.idToNodeMap.get(directInitiatorRequest.requestId) || rootNode;
       const initiators = PageDependencyGraph.getNetworkInitiators(node.record);
       if (initiators.length) {
         initiators.forEach(initiator => {
-          const parentCandidates = networkNodeOutput.urlToNodeMap.get(initiator) || [rootNode];
+          const parentCandidates = networkNodeOutput.urlToNodeMap.get(initiator) || [];
           // Only add the initiator relationship if the initiator is unambiguous
-          const parent = parentCandidates.length === 1 ? parentCandidates[0] : rootNode;
+          const parent = parentCandidates.length === 1 ? parentCandidates[0] : directInitiatorNode;
```

Comment:
we probably should have been doing this all along but now that we're selecting initiators that will be later in load can we add a quick check that the request started *after* the parent?